unlock-app/src/stories/creator/Dashboard.stories.js

```diff
@@ -58,6 +59,7 @@ const locks = {
     maxNumberOfKeys: 240,
     outstandingKeys: 3,
     transaction: '0x1234',
+    owner: account.address,
   },
   '0x9abcdef0a': {
     address: '0x9abcdef0',
```

Comment:
add owner here to preserve the story and fix chromatic

Reply:
Done!